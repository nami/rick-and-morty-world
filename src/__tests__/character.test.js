import React from "react";
import { create } from "react-test-renderer";
import Character from '../components/character'
import { BrowserRouter as Router } from "react-router-dom";

describe("Character component", () => {
  test("Matches the snapshot", () => {
  		const character = create(<Router><Character /></Router>); 
    	expect(character.toJSON()).toMatchSnapshot();
  });
});