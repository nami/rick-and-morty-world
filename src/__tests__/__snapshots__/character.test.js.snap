// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component Matches the snapshot 1`] = `
<div
  className="character box has-background-primary has-text-white animate__animated animate__slideInUp"
>
  <figure
    className="image is-square icons"
  >
    <img
      alt="undefined"
      className="is-rounded"
    />
  </figure>
  <a
    href="/profile/undefined"
    onClick={[Function]}
  >
    <h6
      className="title is-5 name"
    />
  </a>
  <p
    className="subtitle is-6 has-text-white"
  />
</div>
`;
